java/operator/src/main/java/com/instaclustr/cassandra/operator/controller/DataCenterReconciliationController.java

```diff
@@ -116,6 +116,17 @@ private ConfigMapVolumeMount(final String name, final String mountPath, final V1
         }
     }
 
+    private V1PodSpec createPodSpec() {
+        V1PodSpec podSpec = new V1PodSpec();
+        String secret = dataCenterSpec.getImagePullSecret();
+        if (secret != null && secret.length() > 0) {
```

Comment:
This project has Google Guava as a dependency which provides `Strings.isNullOrEmpty(...)`.